### DISCUSSION TITLE
Easy way how to match lat and lon for spatial and forcing data?

### DISCUSSION THREAD

**Discussion by DavidChoi76** [Deltares/Wflow.jl#520] [Q&A]
Hello!!

I have a question how to match lat and lon for spatial and forcing data?
When I downloaded ERA5 data from CDS, the resolution is very coarse. And spatial data such as dem and others have different resolution and range with ERA5 data. Is there an easy way to match lat and lon for spatial and forcing data?

**Comment by JoostBuitink** [comment]
The easiest way to do this, it through the hydromt_wflow Python library. This is a python package that we develop to easily build and update wflow models, also to add forcing to the model (at right lat-lon dimensions). This page contains an example specifically for the forcing: https://deltares.github.io/hydromt_wflow/latest/_examples/update_model_forcing.html

You'd also need to define a data_catalog that describes the ERA5 data that you have. Some information can be found here: https://deltares.github.io/hydromt/v0.10.0/user_guide/data_prepare_cat.html, and you can take some inspiration from this data_catalog: https://github.com/Deltares/hydromt/blob/87525ed2f52998af08523f0a7c169f091d8ab58b/data/catalogs/deltares_data/v0.6.0/data_catalog.yml#L495-L538
